week5.1-ex1.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+
+def files_start_with(directory_path):
+    """ Return list of the files, in the directory at the end of the path, start with deep"""
```

Comment:
![Capture](https://user-images.githubusercontent.com/13467749/160067720-8f6fd19d-4751-4ec0-817a-6592155689ca.JPG)
this is how it should look like . 
add type hints and return type